packages/css/src/components/form/form-groups/search-bar/form-group-search-bar.template.ts

```diff
@@ -0,0 +1,57 @@
+import { FormGroupSearchBar } from '@dso-toolkit/sources';
+
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { infoButtonTemplate } from '../../../info-button/info-button.template';
+import { infoTemplate } from '../../../info/info.template';
+import { searchBarTemplate } from '../../../search-bar/search-bar.template';
+
+export function formGroupSearchBarTemplate(formGroup: FormGroupSearchBar<TemplateResult>): TemplateResult {
+  const errorTextId = `${formGroup.id}-error-text`;
+  const helpTextId = `${formGroup.id}-help-text`;
+  const infoTextId = `${formGroup.id}-info-text`;
+
+  const ariaDescribedBy = [
+    formGroup.info?.fixed ? infoTextId : undefined
+  ]
+    .filter(s => !!s)
+    .join(' ') || undefined;
+
+  return html`
+    <div
+      class="form-group dso-filter ${classMap({ [`dso-${formGroup.state}`]: !!formGroup.state })}"
```

Comment:
Ik begrijp niet zo goed waarom deze form group `dso-filter` heet. Ik heb gezien dat ie ook zo heet in de Nunjucks, maar het klopt niet echt..

* Er is geen styling actief op `.dso-filter`;
* De Form Group heet "Search Bar";

Zullen we 'm hernoemen naar `.form-group.dso-form-group-search-bar` ? Ik stel bewust de modifier `.dso-form-group-search-bar` voor zodat we geen conflicten met de bestaande `.dso-search-bar` krijgen.

_Technisch_ gezien wordt het dan wel een BREAKING change, maar de huidige opzet is zeker niet goed genoeg om in de huidige staat te worden doorgezet. Dan ook in de Nunjucks hernoemen en de selector meenemen in `form-group-search-bar.scss`.

cc: @gofix

Reply:
Ik ben met je eens, laten we deze kans aangrijpen om de boel goed recht te trekken